components/overview/ContentAccordion.vue

```diff
@@ -27,10 +29,13 @@ import AccordionLayout from '~/components/overview/AccordionLayout.vue'
 export default class extends Vue {
   @Prop(Array) elements
 
-  expandedItem = 0
+  open = Array.from({ length: this.elements.length }, (_, i) => i === 0)
```

Comment:
May be `itemStates` or similar is better, something that reminds us this is the list of the state of each item. Annotating the property with the type `boolean[]` also helps figuring out what this is about.